interfaces/redux.ts

```diff
@@ -0,0 +1,10 @@
+import { Currency } from './currency';
+
+export interface ThunkAction {
```

Comment:
[minor] Why is this called ThunkAction? Names should be self explanatory about what they do.